.github/workflows/spellcheck.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: spell and grammer check
+on:
+  pull_request:
+    paths:
+      - "docs"         #(provide name of the file or folder)
+jobs:
+  check-spelling:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Before `steps`, you can add 2 env vars like

```
env:
  SPELL_CHECK_DISABLED: false
  GRAMMAR_CHECK_DISABLED: false
```

And then in each of the 2 steps (spell check and grammar check), you can add if conditions (This helps in disabling checks with just top level boolean values in case this file runs too long in the future with more checks)
`if: ${{ env.SPELL_GRAMMAR_CHECK == 'false' }}`